rapids_dask_dependency/patches/distributed/comm/__rdd_patch_ucx.py

```diff
@@ -561,6 +562,17 @@ def get_connector(self):
         return UCXConnector()
 
     def get_listener(self, loc, handle_comm, deserialize, **connection_args):
+        warnings.warn(
+            "you have requested protocol='ucx', which now defaults to UCXX but "
+            "the package distributed-ucxx is not installed. In the current version "
+            "of Distributed this will fallback to UCX-Py which is now deprecated "
+            "and will be removed in a future release. For now protocol='ucx' will "
+            "fallback to the old UCX-Py library, but for continued use of UCX as "
+            "a Distributed communication backend, please ensure you switch to the "
+            "new distributed-ucxx package. To keep on using UCX-Py for now and "
+            "disable this warning, specify protocol='ucx-old'.",
+            # DeprecationWarning,
```

Comment:
Want `FutureWarning` here?

Reply:
Sure, done in https://github.com/rapidsai/rapids-dask-dependency/pull/116/commits/eebf4fa24283405ecfae955b781dee5853c84ddb.